fs_src/index.html

```diff
@@ -1,595 +1,551 @@
-  <h1 class="">System</h1>
-  <div class="form">
-    <div class="">
-      <div class="form-control">
-        <label>Model:</label>
-        <span id="model"></span>
-      </div>
-      <div class="form-control">
-        <label>Device Id:</label>
-        <span id="device_id"></span>
-      </div>
-      <div class="form-control">
-        <label>Uptime:</label>
-        <span id="uptime"></span>
-      </div>
-      <div class="form-control" id="sys_temp_container" style="display: none">
-        <label>Temperature:</label>
-        <span id="sys_temp"></span>&deg;C
-      </div>
-      <div class="form-control" id="debug_log_container" style="display: none">
-        <label>Debug Log:</label>
-        <label class="switch">
-          <input type="checkbox" id="debug_en">
-          <span class="slider round"></span>
-        </label>
-        <span id="debug_link"><a href="/debug/log?follow=1" style="margin-left: 1em">Log</a></span>
-      </div>
-      <div class="form-control">
-        <label></label>
-        <button class="btn" id="reboot_btn">
-          <label>Reboot</label>
-        </button>
-        <button class="btn" id="reset_btn">
-          <label>Reset</label>
-        </button>
-      </div>
-    </div>
-  </div>
-</div>
-
-<div class="container" id="firmware_container" style="display: none">
-  <a class="helpbadge" href="https://github.com/mongoose-os-apps/shelly-homekit/wiki/Firmware"
-     target="_blank">?</a>
-  <h1 class="">Firmware</h1>
-  <div class="form">
-    <div class="">
-      <div class="form-control">
-        <label>Version:</label>
-        <span id="version"></span>
-      </div>
-      <div class="form-control">
-        <label>Build:</label>
-        <span id="fw_build1"></span>
-      </div>
-      <div class="form-control">
-        <label></label>
-        <span id="fw_build2"></span>
-      </div>
-      <div class="form-control" id="update_container" style="display: none">
-        <label>Update: </label>
-        <button class="btn" id="update_btn">
-          <label>
-            <span id="update_btn_spinner"></span>
-            <span id="update_btn_text">Check</span>
-          </label>
-        </button>
-        <span id="update_status"></span>
-      </div>
-      <div class="form-control" id="revert_to_stock_container" style="display: none">
-        <label>Revert to stock:</label>
-        <button class="btn" id="revert_btn">
-          <label><span id="revert_btn_spinner"></span>Revert</label>
-        </button>
-        <span id="revert_status"></span>
-        <div id="revert_msg" style="margin-left: 9em; display: none">Please consider reporting missing features
-          <a href="https://github.com/mongoose-os-apps/shelly-homekit/issues">on GitHub</a>.
+  <script type="module">
+    // JS code will be extracted back to script.js if work is done
+
+    // DEV MODE
+    // import * as htm_preact_standalone from 'https://unpkg.com/htm/preact/standalone.module.js';

```

Comment:
This is how it looks like in the local dev mode. During the build the make script will inline this js lib file into the html document and it will looks like the following line. For easier development I currently use a static permanent inlined version of preact. We can discuss how we want to handle it, including preact in the repo vs. fetching it on build.